examples/ConvertScancoVolumesToOpenStandardFormats.ipynb

```diff
@@ -0,0 +1,1170 @@
+       " 'ScannerID': '2135',\n",
+       " 'ScannerType': '10',\n",
+       " 'Site': '5',\n",
+       " 'SliceIncrement': '0.036',\n",
+       " 'SliceThickness': '0.036',\n",
+       " 'Version': 'CTDATA-HEADER_V1',\n",
+       " 'origin': array([0., 0., 0.]),\n",
+       " 'spacing': array([0.036, 0.036, 0.036]),\n",
+       " 'direction': array([[1., 0., 0.],\n",
+       "        [0., 1., 0.],\n",
+       "        [0., 0., 1.]])}"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "image_meta = itk.imread('image.mha')\n",
+    "dict(image_meta)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Nifti\n",
+    "\n",
+    "[Nifti](https://nifti.nimh.nih.gov/) is a popular format in the neuroimaging community.\n",
+    "\n",
+    "Nifti:\n",
+    "\n",
+    "- Supports 3D images with pixel with types like `signed short`.\n",
+    "- Image spatial metadata, i.e. `origin`, `spacing`, and `direction`.\n",
+    "- Supports additional key / value metadata."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "itk.imwrite(image, 'image.nii.gz')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'CalibrationData': '45 kVp, 0.5mm Al, BH: 1200mg HA/ccm, Scaling 4096',\n",
+       " 'CreationDate': '5-JUN-2015 11:09:18.880',\n",
+       " 'DataRange': '32767 32767',\n",
+       " 'Energy': '45',\n",
+       " 'ITK_InputFilterName': 'MetaImageIO',\n",
+       " 'Intensity': '0.177',\n",
+       " 'MeasurementIndex': '4937',\n",
+       " 'ModificationDate': '5-JUN-2015 11:09:18.880',\n",
+       " 'MuScaling': '4096',\n",
+       " 'MuWater': '0.7033',\n",
+       " 'NumberOfProjections': '500',\n",
+       " 'NumberOfSamples': '1024',\n",
+       " 'PatientIndex': '78',\n",
+       " 'PatientName': 'COLE-BPBP',\n",
+       " 'ReconstructionAlg': '3',\n",
+       " 'ReferenceLine': '0',\n",
+       " 'RescaleIntercept': '-1000',\n",
+       " 'RescaleSlope': '0.347136',\n",
+       " 'RescaleType': '2',\n",
+       " 'RescaleUnits': 'mg HA/ccm',\n",
+       " 'SampleTime': '400',\n",
+       " 'ScanDistance': '36.864',\n",
+       " 'ScannerID': '2135',\n",
+       " 'ScannerType': '10',\n",
+       " 'Site': '5',\n",
+       " 'SliceIncrement': '0.036',\n",
+       " 'SliceThickness': '0.036',\n",
+       " 'Version': 'CTDATA-HEADER_V1',\n",
+       " 'origin': array([0., 0., 0.]),\n",
+       " 'spacing': array([0.036, 0.036, 0.036]),\n",
+       " 'direction': array([[1., 0., 0.],\n",
+       "        [0., 1., 0.],\n",
+       "        [0., 0., 1.]])}"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "image_nifti = itk.imread('image.mha')\n",
```

Comment:
Should this be `.nii.gz`?

Reply:
Yes, good catch.

Reply:
This revealed that Nifti is not preserving the metadata -- I will remove it from the notebook.